src/app/core/services/fss-search-validator.service.ts

```diff
@@ -39,12 +40,22 @@ export class FssSearchValidatorService {
           }
         }
       }
+      if (!fieldDataType) {
+        if (fssSearchRows[rowId].selectedField === "" || fssSearchRows[rowId].selectedOperator === "") {
+          if (fssSearchRows[rowId].fieldFormControl.touched === false) {
+            fssSearchRows[rowId].fieldFormControl.markAsTouched();
+          }
+        }
+        this.errorMessageTitle = "There is a problem with a Field";
```

Comment:
Please change title to 
`There is a problem with a field`

Reply:
Hi Chris,
I have made the requested change.